migration/i7.1z/oracle/202009161600_IDEMPIERE-3336.sql

```diff
@@ -0,0 +1,132 @@
+;
+
+-- 08/04/2017 11h26min21s BRT
+UPDATE AD_Column SET IsMandatory='Y',Updated=TO_DATE('2017-04-08 11:26:21','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212980
+;
+
+-- Sep 16, 2020, 3:14:12 PM BRT
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,ColumnSpan) VALUES (206507,'Entity Type','Dictionary Entity Type; Determines ownership and synchronization','The Entity Types "Dictionary", "iDempiere" and "Application" might be automatically synchronized and customizations deleted or overwritten.  
+
+For customizations, copy the entity and select "User"!',53285,212980,'Y',40,120,'N','N','N','N',0,0,'Y',TO_DATE('2020-09-16 15:14:12','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2020-09-16 15:14:12','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D','e53eabd4-26e2-4368-ac3e-fe600baa6819','Y',120,2)
+;
+
+-- Sep 16, 2020, 3:14:30 PM BRT
+UPDATE AD_Field SET IsDisplayed='Y', SeqNo=120, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, XPosition=4, IsToolbarButton=NULL,Updated=TO_DATE('2020-09-16 15:14:30','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=206507
+;
+
+-- Sep 16, 2020, 3:14:30 PM BRT
+UPDATE AD_Field SET SeqNo=0, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_DATE('2020-09-16 15:14:30','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=204581
+;
+
+-- defines all records as user customization
+update AD_RelationType set EntityType='U';
+update AD_StatusLineUsedIn set EntityType='U';
+
+-- restore core records as EntityType = 'D'
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200000;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200001;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200002;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200003;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200004;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200005;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200006;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200007;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200008;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200009;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200010;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200011;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 200012;
+update AD_StatusLineUsedIn set EntityType='D' where AD_StatusLineUsedIn_ID = 1000000;
```

Comment:
the usual convention should be EntityType='D' if _ID is < 1000000

Reply:
i'll take care in next script to use this convention. thank you for the review & tip!